apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -459,7 +459,7 @@ type EnvoyLogging struct {
 
 	// AccessLogLevel sets the verbosity level of the access log.
 	//
-	// Values: `info` (default, meaning all requests are logged), `error` and `disabled`.
+	// Values: `info` (default, meaning all requests are logged), `error` (meaning all non-success requests are logged), `critical` (meaning all 5xx requests are logged) and `disabled`
```

Comment:
```suggestion
	// Values: `info` (default, meaning all requests are logged), `error` (meaning all non-success requests, i.e. 300+ response code, are logged), `critical` (meaning all 5xx requests are logged) and `disabled`.
```

Reply:
done in db1d291e83351f7a0da9db8559b4ca6a455a4fec